Stylesheets/components/avtar.css

```diff
@@ -0,0 +1,76 @@
+@import url("/stylesheets/main.css");
+
+.center {
+  margin-left: 0%;
+}
+
+.avtar {
+  margin: 0.3rem;
+  width: 2rem;
+  height: 2rem;
+  border-radius: 50%;
+  position: relative;
+  left: 30%;
+}
+
+.avtar-small {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.avtar-large {
+  width: 3rem;
+  height: 3rem;
+}
+
+.avtar-xlarge {
+  width: 4.5rem;
+  height: 4.5rem;
+}
+
+.avtar-xlarger {
+  width: 5.5rem;
+  height: 5.5rem;
+}
+.avtar-square {
+  border-radius: 5%;
+}
+
+.avtar-container {
+  position: relative;
+  width: fit-content;
+}
+
+.badge {
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  position: absolute;
+  bottom: 20%;
+  right: -25%;
+  border: 1.5px solid white;
+}
+
+.badge-online {
+  background-color: var(--badge-online);
+}
+
+.badge-idle {
+  background-color: var(--badge-idle);
+}
+
+.badge-dnd {
+  background-color: var(--badge-dnd);
+}
```

Comment:
This block is related to badge styles. Shouldn;'t it be in badge.css?

Reply:
it is an extra i added,  I will comment it